app/views/content_items/service_manual_guide.html.erb

```diff
@@ -69,7 +69,7 @@
             data-ga4-set-indexes
             data-ga4-link="<%= {
               event_name: "navigation",
```

Comment:
Hi @andysellick - isn't `event_name` meant to be `select_content`, and the type `contents list`?

Reply:
Checked with Harriet and it should be `event_name` = `select_content` and `type` = `contents list`.

Reply:
So exactly what you said 😁